public/data/info.json

```diff
@@ -0,0 +1,27 @@
+[
```

Comment:
I like this file, but I think the name is a bit wrong. It seems like it's something like `about_page_info.json` or similar.